aws/table_aws_wafv2_web_acl.go

```diff
@@ -375,9 +376,10 @@ func listAssociatedResources(ctx context.Context, d *plugin.QueryData, h *plugin
 			// unsupported region check
 			return nil, nil
 		}
+
 		// Build param
 		param := &cloudfront.ListDistributionsByWebACLIdInput{
-			WebACLId: aws.String(data["ID"]),
+			WebACLId: aws.String(data["Arn"]),
```

Comment:
@ParthaI could you please add a comment as to why we need to pass `Arn` instead of `ID`?